src/shared/components/palette/dynamic-radio-list/write-dynamic-radio-list-field.html

```diff
@@ -0,0 +1,20 @@
+<div class="form-group bottom-30"
+    [ngClass]="{'form-group-error': !dynamicRadioListControl.valid && dynamicRadioListControl.touched}" [id]="id()">
```

Comment:
Pretty sure this should be using the `.dirty` property, not `.touched`.

Reply:
updated to dirty.